e2e/_suites/fleet/services.go

```diff
@@ -497,8 +498,8 @@ func newTarInstaller(image string, tag string) (ElasticAgentInstaller, error) {
 		image:             image,
 		InstallFn:         installFn,
 		installerType:     "tar",
-		logFile:           "elastic-agent.log",
-		logsDir:           "/opt/Elastic/Agent/data/elastic-agent-%s/",
+		logFile:           "elastic-agent-json.log",
+		logsDir:           "/opt/Elastic/Agent/data/elastic-agent-%s/logs/",
```

Comment:
Could you revisit my comment here? https://github.com/elastic/e2e-testing/pull/481#discussion_r533246796

I'd prefer using plain log for readability concerns

Reply:
if it's ok by you then ok, i assumed that for computer processing json is more suitable.

Reply:
Totally agree, although in this case, we are not sending the raw container logs to any other system (it's on our roadmap).

What we are sending to an ES cluster is Jenkins build logs, and because the container log is simply outputted to Jenkins logs, it's stored in the same format as the Jenkins log in our logs cluster.